packages/rpc/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@metamask/test-snap-rpc",
+  "version": "4.1.2",
+  "description": "MetaMask JSON-RPC Permissions Test Snap",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/test-snaps.git"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "license": "ISC",
+  "files": [
+    "dist/",
+    "images/icon.svg",
+    "snap.manifest.json"
+  ],
+  "scripts": {
+    "build": "mm-snap build --eval false",
+    "build:clean": "yarn clean && yarn build",
+    "build:dev": "yarn build",
+    "clean": "rimraf 'dist/*'",
+    "start": "mm-snap watch",
+    "test": "jest --passWithNoTests",
+    "test:watch": "jest --watch",
+    "lint:eslint": "eslint . --cache --ext js,ts",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' --single-quote --ignore-path ../../.prettierignore --no-error-on-unmatched-pattern",
+    "lint": "yarn lint:eslint && yarn lint:misc --check",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
+    "lint:changelog": "yarn auto-changelog validate"
+  },
+  "dependencies": {
+    "@metamask/key-tree": "^6.0.0",
```

Comment:
Doesn't look like all these dependencies are used

Reply:
Also forgot to remove those.